content/en/docs/components/pipelines/quickstart.md

```diff
@@ -3,4 +3,340 @@ title = "Quickstart"
+kfp_client = client.Client(host=endpoint)
+run = kfp_client.create_run_from_pipeline_func(
+    my_pipeline,
+    arguments={
+        'a': 1,
+        'b': 2
+    },
+)
+url = f'{endpoint}/#/runs/details/{run.run_id}'
+print(url)
+```
+
+Let's walk through each part step-by-step.
+
+<!-- TODO: add lightweight component authoring link -->
+First, we begin by creating a lightweight Python component using the `@dsl.component` decorator:
+```python
+@dsl.component
+def addition_component(num1: int, num2: int) -> int:
+    return num1 + num2
+```
+This decorator transforms a Python function into a component that can be used within a pipeline. Type annotations on the parameters and return value are required, as this tells the KFP executor how to serialize and deserialize data as it is passed between components. It also allows the KFP compiler to type check your pipeline.
+
+Second, we create a pipeline using the `@dsl.pipeline` decorator:
+```python
+@dsl.pipeline(name='addition-pipeline')
+def my_pipeline(a: int, b: int, c: int = 10):
+    ...
+```
+Like the component decorator, the pipeline decorator transforms a Python function into a pipeline that can be executed by the KFP backend. The pipeline can have arguments which should also have type annotations. Notice that `c` has a default value of `10`.
+
+Third, we connect components together to form a computational directed acyclic graph (DAG) within the pipeline function body:
+```python    
+add_task_1 = addition_component(num1=a, num2=b)
+add_task_2 = addition_component(num1=add_task_1.output, num2=c)
+```
+
+In this example, we instantiate two different addition tasks from the same `addition_component` by passing arguments to the component function. Component arguments must always be passed as keyword arguments.
+
+The first task takes pipeline parameters `a` and `b` as inputs. The second component takes the output from `add_task_1` as one input using `add_task_1.output` and pipeline parameter `c` as the second input.
+
+Finally, we instantiate a KFP client with the endpoint we obtained in the previous step and submit the pipeline to the backend with the required pipeline arguments:
+
+```python
+endpoint = '<KFP_ENDPOINT>'
+kfp_client = client.Client(host=endpoint)
+run = kfp_client.create_run_from_pipeline_func(
+    my_pipeline,
+    arguments={
+        'a': 1,
+        'b': 2
+    },
+)
+url = f'{endpoint}/#/runs/details/{run.run_id}'
+print(url)
+```
+
+`endpoint` should be the KFP Dashboard URL you obtained in the previous step.
+
+Alternatively, we could have compiled the pipeline to YAML for use at another time:
+
+```python
+from kfp import compiler
+
+compiler.Compiler().compile(pipeline_func=my_pipeline, package_path='pipeline.yaml')
+```
+
+### 4) View the pipeline in the KFP Dashbaord
+Open the URL printed by step three to view the pipeline run on the KFP Dashboard. By clicking on each task node, you can view inputs, outputs, logs, and other task details.
```

Comment:
note that logs are currently not available in v2. It may take a while to be added.